modules/menu.js

```diff
@@ -0,0 +1,40 @@
+// Navigation menu
+const menu = () => {
+  const display = () => {
+    document.getElementById('diplay-list').setAttribute('class', 'non-show');
+    document.getElementById('input-data').setAttribute('class', 'show');
+    document.getElementById('contact').setAttribute('class', 'non-show');
+    document.getElementById('footer').setAttribute('class', 'show');
+  };
+  display();
+
+  const displayList = () => {
+    document.getElementById('diplay-list').setAttribute('class', 'show');
+    document.getElementById('input-data').setAttribute('class', 'non-show');
+    document.getElementById('contact').setAttribute('class', 'non-show');
+  };
+
+  const displayAdd = () => {
```

Comment:
* Currently am unable to add and remove books the nav buttons are not responding too, Kindly fix it 
![image](https://user-images.githubusercontent.com/52497246/220849539-47ccb5ed-5a9a-493b-b9ad-7c5fd03d28c0.png)